SpatialGDK/Documentation/content/get-started/porting-unreal-project-to-gdk.md

```diff
@@ -18,9 +17,17 @@ Before porting your project:
 * Open a terminal window and run the command `spatial update` to ensure your [spatial CLI]({{urlRoot}}/content/glossary#spatial-command-line-tool-cli) installation is up to date. 
 
 ### Terms used in this guide
-`<GameRoot>` - The directory containing your project's `.uproject` file and `Source` directory.  
-`<ProjectRoot>` - The directory containing your `<GameRoot>`.  
-`<YourProject>` - The name of your project's `.uproject` file (for example, `\<GameRoot>\TP_SpatialGDK.uproject`).
+* `<GameRoot>` - The directory containing your project's `.uproject` file and `Source` directory.  
+* `<ProjectRoot>` - The directory containing your `<GameRoot>`.  
+* `<YourProject>` - The name of your project's `.uproject` file (for example, `\<GameRoot>\TP_SpatialGDK.uproject`).
+
+<%(Callout type="tip" message="
+**ShooterGame Reference Project:** 
+
+As you port your own project, you may find it helpful to use our ported [Unreal ShooterGame](https://docs.unrealengine.com/en-us/Resources/SampleGames/ShooterGame) running on the GDK as a reference.
+
+You should already have this project set up as it is included in the `Samples` directory of [the GDK's Unreal Engine fork](https://github.com/improbableio/UnrealEngine). [Here](https://www.youtube.com/watch?v=xojgH7hJgQs&feature=youtu.be) is a video showing what you can expect to see when running it. 
```

Comment:
```suggestion
You should already have this project as it is included in the `Samples` directory of [the SpatialOS Unreal Engine fork](https://github.com/improbableio/UnrealEngine) which you downloaded as part of the _Get started_ steps. <br/>
(If you don't have it, you can get it from the Unreal repository, and if you want to see the game running, there's a [video on youtube](https://www.youtube.com/watch?v=xojgH7hJgQs&feature=youtu.be) to check out.)
```
Slight re-jig to incorporate the link to the GDK and make it clear where and when you would have got it.